docs/advanced-recipes/custom-useatom-hooks.mdx

```diff
@@ -70,4 +70,8 @@ useFocusAtom(anAtom) {
 }
 ```
 
+### CodeSandbox
+
+<CodeSandbox id="y5wef8" />

```

Comment:
Please remove unused code! 
BTW, I see many type issues in the code sandbox, please try fixing them (in case you have problems, try joining the discord so I can help you)

And also issues like useFocusAtom cannot be found!

Reply:
@Aslemammad 
I've sent your a message, please check